charts/kube-state-metrics/templates/_helpers.tpl

```diff
@@ -99,3 +99,28 @@ labelNameLengthLimit: {{ . }}
 labelValueLengthLimit: {{ . }}
 {{- end }}
 {{- end -}}
+
+{{/*
+To help compatibility with other charts which use global.imagePullSecrets.
```

Comment:
please move description to values.yaml.

Reply:
Moved! Also added clarification about the required input for the template.